gnocchi/indexer/alembic/versions/5c4f93e5bb4_mysql_float_to_timestamp.py

```diff
@@ -38,8 +39,21 @@
 
 def upgrade():
     bind = op.get_bind()
+    inspector = Inspector.from_engine(bind)
+
     if bind and bind.engine.name == "mysql":
         op.execute("SET time_zone = '+00:00'")
+
+        previous_cks = {"resource": [], "resource_history": []}
+        for table in ("resource", "resource_history"):
+            existing_cks = [
+                c['name'] for c in inspector.get_check_constraints(table)
+            ]
+            ck_name = "ck_{}_started_before_ended".format(table)
+            if ck_name in existing_cks:
+                op.drop_constraint(ck_name, table, type_="check")
+                previous_cks[table].append(ck_name)
+
```

Comment:
not sure why this is needed for. Might be great to write it in the commit message.

Reply:
This fixes that the migration is broken for MySQL 8. The migration drops columns that is still being referenced by check constraints and you get something like.

ERROR 3959 (HY000): Check constraint 'ck_resource_started_before_ended' uses column 'started_at', hence column cannot be dropped or renamed.

Reply:
Ok, though the PR mentions "SQLAlchemy 1.4 and Python 3.8", so there are way too many changes for this scope.

I mean, then it's fine if you can't split that into multiple PR, but at least you should write a proper commit message, or body, whatever, for the future-you re-reading that in 2 years 😉

Reply:
Yeah, sorry about that. Since CI is usually broken I just combine all the things. I'll make sure to split everything up next time!

Reply:
Updated PR with some more details for future reference.